src/main/java/nl/tudelft/fishy/EndBoss.java

```diff
@@ -73,37 +77,26 @@ private static EndBoss generateBoss() {
 
     // Generate the end boss, depending on which side of the screen it spawns.
     if (isLefty) {
-      // Get the image for the end boss and its respective height and width.
-      Image fishImage = new Image(leftImageFileName);
-      fishImage = new Image(rightImageFileName, fishImage.getWidth() * imgSizeMultiplier,
-          fishImage.getHeight() * imgSizeMultiplier, true, true);
-      int fishImageWidth = (int) fishImage.getWidth();
-      int fishImageHeight = (int) fishImage.getHeight();
-      sprite = new Sprite(fishImage, boundingBox);
-      boundingBox.setHeight(fishImageHeight);
-      boundingBox.setWidth(fishImageWidth);
-      boundingBox.setX(-2000);
-      boundingBox.setY(-2000);
-      sprite.setBoundingBox(boundingBox);
-     
-      return new EndBoss(speed, isLefty, sprite);
+      imageFileName = rightImageFileName;
```

Comment:
Yes, it is intended to do that, since right file name means the image of the fish is heading towards right. But is lefty means that the fish is coming from left. It was also confusing for me at first, but I tested it through running the game.
< naming